tests/test_data_submission.py

```
@@ -548,18 +556,18 @@
     assert 1 == fingerprint["id"]
     assert 1 == fingerprint["track_id"]
 
-    query = tables.fingerprint.select(tables.fingerprint.c.id == 1)
-    fingerprint = fingerprint_db.execute(query).fetchone()
+    query = select(tables.fingerprint).where(tables.fingerprint.c.id == 1)
+    fingerprint = dict(fingerprint_db.execute(query).one()._mapping)
     assert fingerprint is not None
     assert 1 == fingerprint["track_id"]
```

Ruff (0.8.2)

Yoda condition detected

Rewrite as `fingerprint["track_id"] == 1`

(SIM300)